webapi/src/main/scala/org/knora/webapi/responders/v2/StandoffResponderV2.scala

```diff
@@ -181,8 +182,8 @@ final case class StandoffResponderV2(
         } else {
           for {
             response <-
-              messageRelay
-                .ask[SipiGetTextFileResponse](
+              sipiServiceLive
+                .getTextFileRequest(
```

Comment:
*praise*: Nice, one location less where the `MessageRelay` is used

Reply:
👍